src/test/java/org/ebml/matroska/MatroskaFileWriterTest.java

```diff
@@ -117,6 +117,65 @@ public void testMultipleTracks() throws Exception
     testDocTraversal();
   }
 
+  @Test
+  public void testMultipleTracksToFillVoidArea() throws Exception
+  {
+    final MatroskaFileWriter writer = new MatroskaFileWriter(ioDW);
+
+    for (int i = 0; i < 40; i++)
+    {
+      final MatroskaFileTrack nextTrack = new MatroskaFileTrack();
+      nextTrack.setTrackNo(i + 2);
```

Comment:
The `+2` is only necessary to make line 135 work? Can we simplify?

Reply:
Actually no idea why it is there. I dropped it.